app/src/db/migrations/env.py

```diff
@@ -1,24 +1,12 @@
 import logging
-import sys
 from typing import Any
 
+import alembic.context as context
 import sqlalchemy
-from alembic import context
 
-# Alembic cli seems to reset the path on load causing issues with local module imports.
-# Workaround is to force set the path to the current run directory (top level src folder)
-# See database migrations section in `./database/database-migrations.md` for details about running migrations.
-sys.path.insert(0, ".")  # noqa: E402
-
-# Load env vars before anything further
-from src.util.local import load_local_env_vars  # noqa: E402 isort:skip
-
-load_local_env_vars()
```

Comment:
How do the env vars get loaded locally? I thought that was done when running Flask commands, but this doesn't look to run via Flask.

Reply:
good question. i have to confess that was prioritizing getting database integration working, so i intentionally let running on host machine break. i was gonna work on that soon with https://github.com/navapbc/template-application-flask/issues/136

i'm currently thinking of having developers install direnv, and having an `app/.envrc` file that looks like

```
source local.env
```

and then whenever you `cd` into `app` direnv will automatically get all the env vars from local.env into your shell, and when you `cd` out of the directory direnv will automatically unload those env vars.

the benefit of this is that local developers can also override vars in `local.env` without risking checking those in by adding the overrides to `.envrc` e.g.

```
source local.env
export DB_HOST=localhost
```